tests/test_paragraph.py

```diff
@@ -61,3 +69,44 @@ def test_Paragraph_linebreak(paragraph_builder, textlayout_text_style, textlayou
     paragraph = builder.Build()
     paragraph.layout(300)
     assert (paragraph.Height > 0) and (paragraph.Height > paragraph.LongestLine * 2)
+
+
```

Comment:
I would suggest adding a parameter testing here for say, 0.5, 1.0 and 2.0. How does it behave if you put zero in?

Obviously the assert at the end will need to be adjusted accordingly... (for 0.5 and 0).